Assets/HoloToolkit/Utilities/Scripts/FadeScript.cs

```diff
@@ -46,7 +48,14 @@ void Start()
 #endif
 
             currentState = FadeState.idle;
-            fadeMaterial = GetComponentInChildren<MeshRenderer>().material;
+            if (FadeSharedMaterial)
+            {
+                fadeMaterial = GetComponentInChildren<MeshRenderer>().sharedMaterial;
+            }
+            else
+            {
+                fadeMaterial = GetComponentInChildren<MeshRenderer>().material;
```

Comment:
`.material` is bad unless we're destroying the fade material later.

Reply:
This PR will then address both #1112 and #1113.
Not my original intent.